starter-code/index.html

```diff
@@ -3,40 +3,90 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link href="./css/main.css" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- nav bar -->
 
-What’s on Spotify?
+<div class="nav-bar">
+  <div class="logo"><img src="./images/spotify-logo.png" alt="logo"></div>
+<div class="links">
+  <ul>
+    <li>Premium</li>
+    <li>Discover</li> 
+    <li>Help</li> 
+    <li>Download</li>
+  </ul>
+</div>
+</div>
+<div class="clear"></div>
+<!-- music for everyone -->
 
-Millions of Songs
-There are millions of
-songs on Spotify
+<div class="everyone">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.
+Listen to the right music, wherever you are.</p>
+</div>
 
-HD Music
-Listen to music as if you
-were listening live
+<!-- whats on spodify -->
 
-Stream Everywhere
-Stream music on your
+<div class="whats-on">
```

Comment:
I do not like your choice for this div name... instead of something so general like `"whats-on" , a better choice would be something like "options-wrapper" , or "options-container", or "middle-links" or "middle-row" ..... the point is , do not pick to name a div class by choosing some words that are part of that section, like you chose "whats-on" because the heading of that section is called "what's on Spotify" .  The reason to avoid doing this, is because usually we are going to reuse div names throughout our code, so if we were to stick with something so specific to that section, like sticking with calling it "whats-on", then it would be messy to reuse this name in another part of your code , since there probably only be one section using that heading "whats-on" , if we were to pick a better name then it would fit to reuse it in other parts of the code.